lib/openxml/drawingml/properties/custom_geometry.rb

```diff
@@ -0,0 +1,18 @@
+module OpenXml
+  module DrawingML
+    module Properties
+      class CustomGeometry < SimplePropertyContainerProperty
+        namespace :a
+        tag :custGeom
+
+        property :shape_adjustment_list
+        property :geometry_guide_list
+        # property :adjustable_handle_list # ahLst
+        # property :connection_list # cxnLst
+        # property :rectangle # rect
+        property :path_list, required: true
```

Comment:
Are these just to remember what else could be implemented for this property?

Reply:
(In other places, I've seen you use `# TODO: add `sp3d` property` — which is clearer what's meant by it)

Reply:
I can add the explicit `TODO: add`, since that's exactly what this is meant to indicate.